docs/usage.md

```diff
@@ -153,7 +153,7 @@ Now the `env` preset will only load transformation plugins for features that are
 
 The [@babel/polyfill](polyfill.md) module includes [core-js](https://github.com/zloirock/core-js) and a custom [regenerator runtime](https://github.com/facebook/regenerator/blob/master/packages/regenerator-runtime/runtime.js) to emulate a full ES2015+ environment.
 
-This means you can use new built-ins like `Promise` or `WeakMap`, static methods like `Array.from` or `Object.assign`, instance methods like `Array.prototype.includes`, and generator functions (provided you use the [regenerator](plugin-transform-regenerator.md) plugin). The polyfill adds to the global scope as well as native prototypes like `String` in order to do this.
+This means you can use new built-ins like `Promise` or `WeakMap`, static methods like `Array.from` or `Object.assign`, instance methods like `Array.prototype.includes`, and generator functions (provided you as the [regenerator](plugin-transform-regenerator.md) plugin). The polyfill adds to the global scope as well as native prototypes like `String` in order to do this.
```

Comment:
> provided you as the regenerator plugin

Happy to improve this so that there's less confusion, but this change makes less sense to me?

We aren't providing it, the original "provided you use the regenerator plugin" was more "when used alongside the regenerator plugin".

Reply:
so sorry about that. my bad. 
well, the existing sentence doesn't make sense at all.
my suggestion is: "provided to you by the regenerator plugin" as it has a passive structure.

Reply:
I think that here by "provided" it means "assuming that"

Reply:
> and generator functions (when used alongside the [regenerator](plugin-transform-regenerator.md) plugin)

?

Reply:
looks good. now makes sense. 
we need to update polyfill depreciation as well.